.github/workflows/ci-core.yml

```diff
@@ -227,6 +227,14 @@ jobs:
       - name: Check test results
         if: needs.core.result != 'success'
         run: exit 1
+      - name: Notify Slack
+        if: failure() && ${{ matrix.cmd }} == 'go_core_race_tests' && github.event.schedule != ''
```

Comment:
@HenryNguyen5 This doesn't seem to be working. This step is executing for successful, non-race, non-scheduled runs :facepalm:
Is something wrong with the `&&` ?

Reply:
#10221